src/components/dms/ConvoMenu.tsx

```diff
@@ -165,12 +166,9 @@ let ConvoMenu = ({
                 icon={profile.viewer?.blocking ? PersonCheck : PersonX}
               />
             </Menu.Item>
-            <Menu.Item
-              label={_(msg`Report account`)}
-              onPress={() => {}}
-              disabled>
+            <Menu.Item label={_(msg`Report chat`)} onPress={reportControl.open}>
               <Menu.ItemText>
-                <Trans>Report account</Trans>
+                <Trans>Report chat</Trans>
```

Comment:
Maybe should use "message" for this. Even if we call the service "Chat" I think message sounds better here.

Reply:
Agreed